mne/preprocessing/ecg.py

```diff
@@ -311,23 +309,19 @@ def create_ecg_epochs(raw, ch_name=None, event_id=999, picks=None,
                 ecg_raw.info[k] = v
         raw.add_channels([ecg_raw])
 
-    if picks is None and not keep_ecg:
-        picks = pick_types(raw.info, meg=True, eeg=True, ecg=False,
-                           ref_meg=False)
-    elif picks is None and keep_ecg and not_has_ecg:
-        picks = pick_types(raw.info, meg=True, eeg=True, ecg=True,
-                           ref_meg=False)
-    elif keep_ecg and not_has_ecg:
-        picks_extra = pick_types(raw.info, meg=False, eeg=False, ecg=True,
-                                 ref_meg=False)
-        picks = np.concatenate([picks, picks_extra])
-
+    if keep_ecg:
+        if has_ecg:
+            raise ValueError('keep_ecg can be True only if the ECG channel is '
+                             'created synthetically.')
```

Comment:
You should add to the docstring for this kwarg that it *must* be False in cases where a synthetic channel isn't used. Otherwise people might reasonably assume that for data already containing ECG where they use ECG for this, `keep_ecg=True` is okay.